aws/table_aws_config_conformance_pack.go

```diff
@@ -24,55 +24,48 @@ func tableAwsConfigConformancePack(_ context.Context) *plugin.Table {
 		},
 		GetMatrixItem: BuildRegionList,
 		Columns: awsRegionalColumns([]*plugin.Column{
-			{
-				Name:        "arn",
-				Description: "Amazon Resource Name (ARN) of the conformance pack.",
-				Type:        proto.ColumnType_STRING,
-				Transform:   transform.FromField("ConformancePackArn"),
-			},
 			{
 				Name:        "name",
 				Description: "Name of the conformance pack.",
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("ConformancePackName"),
 			},
 			{
-				Name:        "conformance_pack_id",
-				Description: "ID of the conformance pack.",
+				Name:        "conformance_pack_arn",
```

Comment:
Following Terraform, we should keep this as `arn` (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/config_conformance_pack#arn). Also, speaking with @johnsmyth , we normally do name any ARN fields as `arn` (we should clarify this in our [developer docs for naming](https://steampipe.io/docs/develop/standards) though).